CefSharp/IWebBrowser.cs

```diff
@@ -54,11 +54,11 @@ public interface IWebBrowser : IDisposable
         void Load(string url);
 
         /// <summary>
-        /// Registers and loads a <see cref="ResourceHandler"/> that represents the HTML content.
+        /// Registers and loads a <see cref="ResourceHandlerFactory"/> that represents the HTML content.
```

Comment:
Possibly this should stay the same as well? (And the next few instances).

Feel free to improve the comments as you see fit.